nbclassic/notebookapp.py

```diff
@@ -195,6 +195,7 @@ def _prepare_templates(self):
             base_dir, 'nbclassic/i18n'), fallback=True)
         self.jinja2_env.install_gettext_translations(nbui, newstyle=False)
         self.jinja2_env.globals.update(nbclassic_path=nbclassic_path)
+        self.jinja2_env.globals.update(nbclassic_tree=url_path_join(self.serverapp.base_url, nbclassic_path(), "tree"))
```

Comment:
Is there an indentation mismatch?

Reply:
This is a whole new line. I am seeing 8 spaces before just like the line above,

Reply:
Oh yes, this was prolly a bad display in my browser. closing.